web/service/src/main/java/org/mobilitydata/gtfsvalidator/web/service/CreateJobBody.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.mobilitydata.gtfsvalidator.web.service;
+
+public class CreateJobBody {
+
+  private String countryCode;
+  private String url;
```

Comment:
Am I correct that `url` is optional but `countryCode` is not?

Reply:
Currently, url is optional and country code defaults to "US" if not provided.

I'm working on updating the logic so both are optional.

Reply:
They are now both optional.